jest.spec14.config.js

```
@@ -0,0 +1,167 @@
+ * Jest Configuration for SPEC 14 - QA Test Suite Integral
+ * 
+ * Specialized configuration for comprehensive testing:
+ * - E2E scenarios with extended timeouts
+ * - Contract tests with mock safety
+ * - Idempotency tests with database isolation
+ * - Tier validation with billing mocks
+ */
+
+module.exports = {
+  // Test environment
+  testEnvironment: 'node',
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)